packages/invoice-template/src/PolishInvoiceTemplate/fontsConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { FontsConfiguration } from "@leancodepl/pdf-renderer";
+
+export const FontRegular = Symbol("FontRegular");
+export const FontBold = Symbol("FontBold");
+
+const moduleConfig = ({ regular, bold }: { regular: string | Buffer; bold: string | Buffer }): FontsConfiguration => ({
+    [FontRegular]: {
+        fontFamily: "Font",
+        fontFile: regular,
+        fontStyle: "normal",
+        fontWeight: 400,
+    },
+    [FontBold]: {
+        fontFamily: "Font",
+        fontFile: bold,
+        fontStyle: "normal",
+        fontWeight: 600,
+    },
+});
+
+const pdfRendererConfig = [FontRegular, FontBold];
+
+export const polishInvoiceFontsConfig = {
```

Comment:
Why not just export directly and then let decide the user how they want to use it?
Whether they want 
```ts
import * as polishInvoiceFontsConfig from "x";
// or
import { moduleConfig, pdfRendererConfig } from "x";
```